src/modules/keyboardmanager/ui/KeyDropDownControl.h

```diff
@@ -91,9 +89,9 @@ class KeyDropDownControl
     // Function to add a shortcut to the UI control as combo boxes
     static void AddShortcutToControl(Shortcut shortcut, Grid table, StackPanel parent, KeyboardManagerState& keyboardManagerState, const int colIndex, std::vector<std::unique_ptr<KeyDropDownControl>>& keyDropDownControlObjects, RemapBuffer& remapBuffer, StackPanel controlLayout, TextBox targetApp, bool isHybridControl, bool isSingleKeyWindow);
 
-    // Get keys code list depending if Disable is in dropdown
-    static std::vector<DWORD> GetKeyCodeList(bool isShortcut, bool renderDisable);
-
     // Get keys name list depending if Disable is in dropdown
-    static std::vector<std::wstring> GetKeyNameList(bool isShortcut, bool renderDisable);
+    static std::vector<std::pair<DWORD,std::wstring>> GetKeyNameList(bool isShortcut, bool renderDisable);
```

Comment:
[nit] since it has both the key code and the key name, we can rename this to GetKeyList